doc/guides/connection_management.rst

```diff
@@ -234,144 +225,196 @@ Example:
+`3.0`.
+
+  >>> print(nest.GetKernelStatus('num_connections'))
+  9
+
+If you want to connect with different :ref:`receptor types <receptor-types>`, you can do the following:
+
+  ::
+
+    src = nest.Create('iaf_psc_exp_multisynapse', 7)
+    trgt = nest.Create('iaf_psc_exp_multisynapse', 7, {'tau_syn': [0.1 + i for i in range(7)]})
+
+    syn_spec = nest.CollocatedSynapses({'weight': 5.0, 'receptor_type': 2},
+                                       {'weight': 1.5, 'receptor_type': 7})
+
+    nest.Connect(src, trgt, 'one_to_one', syn_spec=syn_spec)
+
+    conns = nest.GetConnections()
+    print(conns.get())
+
+You can see how many synapse parameters you have by doing `len()` on your `CollocatedSynapses` object:
+
+  >>> len(syn_spec)
+  2
+
 
 .. _dist_params:
 
 Distributed parameters
-~~~~~~~~~~~~~~~~~~~~~~~
-
-**Distributed** parameters are initialized with yet another dictionary
-specifying the 'distribution' and the distribution-specific parameters,
-whose specification is optional.
-
-Available distributions are given in the ``rdevdict``, the most common ones
-are:
-
-Distributions Keys::
-
- - 'normal', 'mu', 'sigma'
- - 'normal_clipped', 'mu', 'sigma', 'low ', 'high'
- - 'normal_clipped_to_boundary', 'mu', 'sigma', 'low ', 'high'
- - 'lognormal', 'mu',  'sigma'
- - 'lognormal_clipped', 'mu', 'sigma', 'low', 'high'
- - 'lognormal_clipped_to_boundary', 'mu', 'sigma', 'low', 'high'
- - 'uniform', 'low', 'high'
- - 'uniform_int', 'low', 'high'
- - 'binomial', 'n', 'p'
- - 'binomial_clipped', 'n', 'p', 'low', 'high'
- - 'binomial_clipped_to_boundary', 'n', 'p', 'low', 'high'
- - 'gsl_binomial', 'n', 'p'
- - 'exponential', 'lambda'
- - 'exponential_clipped', 'lambda', 'low', 'high'
- - 'exponential_clipped_to_boundary', 'lambda', 'low', 'high'
- - 'gamma', 'order', 'scale'
- - 'gamma_clipped', 'order', 'scale', 'low', 'high'
- - 'gamma_clipped_to_boundary', 'order', 'scale', 'low', 'high'
- - 'poisson', 'lambda'
- - 'poisson_clipped', 'lambda', 'low', 'high'
- - 'poisson_clipped_to_boundary', 'lambda', 'low', 'high'
-
-Example
+~~~~~~~~~~~~~~~~~~~~~~
+
+**Distributed** parameters are given as ``nest.Parameter`` objects that represent
+values drawn from random distributions. These distributions can either be based on
+spatial node parameters, on default values, or on constant distribution values
+you provide. It is possible to combine the parameters to create distributions
+tailor made for your needs.
+
+The following parameters and functionalities are provided:
+
+- Random parameters
+- Spatial parameters
+- Spatially distributed parameters
+- Mathematical functions
+- Clipping, redrawing, and conditional parameters
+
+For more information, check out the documentation on the different
+:doc:`PyNEST api's <../ref_material/pynest_apis>`, or the section on parametrization
+in our :doc:`nest2_to_nest3/nest2_to_nest3_overview` guide.
```

Comment:
Should we add a ``:ref:`` to the section?

Reply:
Yes! I wanted to do that, I just didn't know how, I will update :)